.github/workflows/release_nightly_on_schedule.yml

```diff
@@ -31,9 +31,9 @@ jobs:
 
   notify:
     name: Notify Lark via webhook
-    needs: release
+    needs: build-n-publish
     runs-on: ubuntu-latest
-    if: ${{ always() }}
+    if: github.repository == 'hpcaitech/ColossalAI'
```

Comment:
You need to keep `always()` so that notification is triggered even if `build-n-publish` fails. The correct code is given below.

```
if: ${{ always() }} && github.repository == 'hpcaitech/ColossalAI'
```

Reply:
Woo, I see.